ipam/store.go

```diff
@@ -50,3 +51,47 @@ type IpamVm struct {
 	Id uint64 `sql:"AUTO_INCREMENT"`
 	//	IpamHostId sql.NullString
 }
+
+type ipamStore struct {
+	common.DbStore
+}
+
+func (ipamStore *ipamStore) addVm(stride uint, vm *Vm) error {
+	tx := ipamStore.DbStore.Db.Begin()
```

Comment:
Isn't "Db" and therefore "Db.Begin()" really a lower-level implementation detail? Should users of DbStore need to know this? What if at some point we replace this with a NoSQL store, where we won't have gorm? Maybe DbStore itself should offer 'begin', 'end', 'commit', whatever, so that it can effectively hide those lower level implementation choices?